README.md

```diff
@@ -28,6 +28,8 @@ Each AWS resource has an associated module that will create Lightstep dashboards
 * __Application ELB__ (module: [`applicationelb-dashboard`](https://github.com/lightstep/terraform-lightstep-aws-dashboards/tree/master/modules/applicationelb-dashboard))
```

Comment:
This has been reviewed. And necessary changes have been made to pass github actions CI and local terraform apply.